spec.md

```diff
@@ -83,7 +83,7 @@ OpenTracing supports a number of different platforms, and of course the per-plat
 
 The `Span` interface must have the following capabilities:
 
-- Finish the (already-started) `Span`.  Finish should be the last call made to any span instance, and to do otherwise leads to undefined behavior. **(py: `finish`, go: `Finish`)**
+- Finish the (already-started) `Span`.  Finish should be the last call made to any span instance, and to do otherwise leads to undefined behavior. **(py: `finish`, go: `Finish`)** Finish is not guaranteed to be called in circumstances such as a bug, node failure or unfinished `Span`. Implementations should clearly document the durability guarantees they provide in such cases.
```

Comment:
I very much like the spirit of this... Would you be okay with the following wording instead?

```
Some implementations may record information about active `Span`s before they are `Finish`ed (e.g., for long-lived `Span` instances), or `Finish` may never be called due to host process failure or programming errors. Implementations should clearly document the `Span` durability guarantees they provide in such cases.
```